docs/_data/sidebar.yml

```diff
@@ -4,17 +4,9 @@
   baseUrl: /getting-started/
   children:
     - title:
-        en: Installation
+        en: Salesforce Extensions for Desktop
         ja: インストール
       url: /getting-started/install
-    - title:
```

Comment:
Same question as with `java-setup.md` should we delete these files (first-project.md & migrate-from-forcecom-ide.md`) or how are we linking to them now ?

Reply:
@lcampos - I think the only links to them are via the sidebar.

Reply:
@lcampos - ok, I deleted them.